homeassistant/components/generic/camera.py

```diff
@@ -75,25 +75,74 @@ async def async_setup_platform(
 ) -> None:
     """Set up a generic IP Camera."""
 
-    await async_setup_reload_service(hass, DOMAIN, PLATFORMS)
+    _LOGGER.warning(
+        "Loading generic IP camera via configuration.yaml is deprecated, "
+        "it will be automatically imported.  Once you have confirmed correct "
+        "operation, please remove 'generic' (IP camera) section(s) from "
+        "configuration.yaml"
+    )
+    imagesource = config.get(CONF_STILL_IMAGE_URL)
```

Comment:
Please separate words with an underscore. Aka image_source

Reply:
Ok.  I've shortened to `image` because there were lots of line-wraps otherwise. 832ced9